seatunnel-connectors-v2/connector-redis/src/main/java/org/apache/seatunnel/connectors/seatunnel/redis/source/RedisSourceFactory.java

```diff
@@ -54,7 +54,9 @@ public OptionRule optionRule() {
                         RedisSourceOptions.HASH_KEY_PARSE_MODE,
                         RedisBaseOptions.AUTH,
                         RedisBaseOptions.USER,
-                        RedisBaseOptions.KEY)
+                        RedisBaseOptions.KEY,
+                        RedisSourceOptions.READ_KEY_ENABLED,
+                        RedisSourceOptions.SINGLE_FIELD_NAME)
```

Comment:
suggest mark RedisSourceOptions.SINGLE_FIELD_NAME is required when data_type = key

Reply:
my bad, when data_type = key. SINGLE_FIELD_NAME should not be required.